include/beast/core/handler_alloc.hpp

```diff
@@ -55,6 +55,12 @@ class handler_alloc
     using value_type = T;
     using is_always_equal = std::true_type;
 
+    template<class U>
+    struct rebind
+    {
+        using other = handler_alloc<U, CompletionHandler>;
+    };
+
```

Comment:
interesting thanks.  I'll bet that is a gcc version prior to a full implementation of `allocator_traits`.